Dockerfile.base

```diff
@@ -125,4 +125,7 @@ RUN apt update &&\
 	apt-get install -y php7.4-cli php7.4-xml php7.4-mbstring docker-compose &&\
 	composer install
 
+# Install sentry-cli.
+RUN wget -O - https://sentry.io/get-cli/ | bash
```

Comment:
For some reason, when I did this as a standalone step in the job, it wasn't available in subsequent steps. 

If this works, I think it'd be better in general because it's one less thing to do on every run :)